core/src/file/cas/checksum.rs

```diff
@@ -1,34 +1,34 @@
-use anyhow::Result;
 use data_encoding::HEXLOWER;
 
 use ring::digest::{Context, SHA256};
 use std::convert::TryInto;
 use std::fs::File;
-use std::io::{BufReader, Read};
 
-#[cfg(target_family = "unix")]
-use std::os::unix::prelude::FileExt;
-
-#[cfg(target_family = "windows")]
-use std::os::windows::prelude::*;
+use std::io;
 use std::path::PathBuf;
 
 static SAMPLE_COUNT: u64 = 4;
 static SAMPLE_SIZE: u64 = 10000;
 
-fn read_at(file: &File, offset: u64, size: u64) -> Result<Vec<u8>> {
+fn read_at(file: &File, offset: u64, size: u64) -> Result<Vec<u8>, io::Error> {
 	let mut buf = vec![0u8; size as usize];
 
 	#[cfg(target_family = "unix")]
-	file.read_exact_at(&mut buf, offset)?;
+	{
+		use std::os::unix::prelude::FileExt;
```

Comment:
For consistency you could include the whole prelude
```suggestion
		use std::os::unix::prelude::*;
```